pennylane/devices/qubit/initialize_state.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions to prepare a state."""
+
+from typing import Iterable, Union
+
+import pennylane as qml
+from pennylane import numpy as np
+
+
+def create_initial_state(
+    wires: Union[qml.wires.Wires, Iterable],
+    prep_operation: qml.operation.StatePrep = None,
+    like: str = "autograd",
```

Comment:
Probably best to start with `"numpy"`.

Reply:
my intention was to [do the same as QNode](https://github.com/PennyLaneAI/pennylane/blob/07b3e6e3cf9ec05c5f7e35bcb784825ccb04ca6d/pennylane/qnode.py#L355), but I'm happy to use `numpy` if we're ready to start moving away from autograd.